src/panel/layouts.js

```diff
@@ -31,20 +35,23 @@ export function getMapPaddings({ isMobile, isDirectionsActive }) {
   };
 }
 
-export function getVisibleBbox(mb) {
+export function getVisibleBbox(mb, isIframe) {
 
   const bbox = mb.getBounds();
   let ne = bbox.getNorthEast();
   let sw = bbox.getSouthWest();
   const ne_canvas = mb.project(ne);
   const sw_canvas = mb.project(sw);
 
-  if (isMobileDevice()) {
-    // On mobile, compute a bbox that excludes the header's height
-    ne_canvas.y += 65;
-  } else {
-    // On desktop, compute a bbox that excludes the left panel's width
-    sw_canvas.x += DESKTOP_PANEL_WIDTH + ADDITIONAL_PADDING / 2;
+  // iframe: no offset
+  if (!isIframe) {
+    if (isMobileDevice()) {
+      // On mobile, compute a bbox that excludes the header's height
+      ne_canvas.y += 65;
```

Comment:
can we define an constant like for desktop ?

Reply:
to replace the 65? yes, though it may be off-topic for this PR, no? (it was already like that)

Reply:
ah, yes ok